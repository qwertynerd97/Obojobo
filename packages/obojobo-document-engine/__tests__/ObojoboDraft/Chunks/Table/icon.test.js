import React from 'react'
import renderer from 'react-test-renderer'

import Icon from 'ObojoboDraft/Chunks/Table/icon'

describe('Table Icon', () => {
	test('Icon', () => {
		const component = renderer.create(<Icon />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
