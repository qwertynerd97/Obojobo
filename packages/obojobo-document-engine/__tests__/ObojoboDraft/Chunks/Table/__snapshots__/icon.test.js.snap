// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table Icon Icon 1`] = `
<svg
  data-name="Layer 1"
  id="Layer_1"
  viewBox="0 0 19.5 14.75"
  xmlns="http://www.w3.org/2000/svg"
>
  <defs>
    <style>
      .table{opacity:0.55;}.line{fill:none;stroke:#231f20;stroke-miterlimit:10;}
    </style>
  </defs>
  <title>
    table
  </title>
  <g
    className="table"
  >
    <rect
      className="line"
      height="13.75"
      width="18.5"
      x="0.5"
      y="0.5"
    />
    <line
      className="line"
      x1="9.75"
      x2="9.75"
      y1="0.5"
      y2="14.25"
    />
    <line
      className="line"
      x1="0.5"
      x2="19"
      y1="9.67"
      y2="9.67"
    />
    <line
      className="line"
      x1="0.5"
      x2="19"
      y1="5.08"
      y2="5.08"
    />
  </g>
</svg>
`;
